### PR TITLE
Review

### PR DESCRIPTION
**Pull request by vrothberg**
Throwing it out to see if CI is kicking in.

### CODE REVIEW

**Comment by vrothberg** [comment]
@baude, it works like a charm. Really good job!

The commits here are just minor janitorial things.

File: plugins/meta/dnsname/files.go [hunk 1]
```diff
@@ -4,13 +4,14 @@ import (
...
 	"fmt"
-	"github.com/coreos/go-iptables/iptables"
 	"io/ioutil"
 	"net"
 	"os"
 	"strings"
 	"text/template"
 
+	"github.com/coreos/go-iptables/iptables"
+
```
Comment (rhatdan) [inline plugins/meta/dnsname/files.go]: Remove open line.

File: plugins/meta/dnsname/main.go [hunk 2]
```diff
@@ -195,10 +202,10 @@ func cmdCheck(args *skel.CmdArgs) error {
 		conffiles = append(conffiles, f.Name())
 	}
 	if !stringInSlice("addnhosts", conffiles) {
-		return errors.New("addnhost file missing from configuration")
+		return fmt.Errorf("addnhost file missing from configuration")
```
Comment (rhatdan) [inline plugins/meta/dnsname/main.go]: Why not use errors.Errorf("")

**Comment by baude** [comment]
LGTM.  Fix the stuff from Dan and Ill merge.  Thanks bigtime

File: plugins/meta/dnsname/main.go (hunk 2 above)
Comment (vrothberg) [reply plugins/meta/dnsname/main.go]: Oooh, I didn't know that existed. I only knew about `errors.New()` until now. Thanks Dan!

**Comment by vrothberg** [comment]
@baude, done :heavy_check_mark:

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**